src/inference/graph/builder.py

```
@@ -1,146 +1,135 @@
+        return "approval_wait" if needs_approval else "tools"
+
+    return route_agent
 
 
 def build_govon_graph(
-    planner_adapter: PlannerAdapter,
-    executor_adapter: ExecutorAdapter,
+    *,
+    llm,
+    tools: list,
     session_store: "SessionStore",
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)